src/main/kotlin/blackjack/model/Score.kt

```diff
@@ -9,15 +9,26 @@ value class Score(val value: Int) {
         }
     }
 
-    fun isLose(): Boolean {
+    fun isBust(): Boolean {
         return value > MAX_SCORE_CRITERIA
     }
 
+    fun isWinThan(that: Score): Boolean {
+        if (isBust()) {
+            return false
+        } else if (that.isBust()) {
+            return true
+        } else {
+            return value >= that.value
+        }
+    }
+
```

Comment:
딜러의 결과와 관계없이 플레이어가 bust인 경우 무조건 플레이어가 패배한다는 요구사항이 잘 반영된 것 같습니다 👍 
제안드리고 싶은게 두 가지 있습니다.

1. 지금은 다음처럼 isWinThan을 호출하고 있기에 위의 요구사항이 만족했지만 (딜러의 점수).isWinThan(플레이어의 점수)와 같이 호출되면 위 요구사항이 충족되지 않습니다. 어떻게 개선해볼 수 있을까요?
```kotlin
    private fun isPlayerWin(player: Player, dealer: Player): Boolean {
        return player.cards.optimalScore().isWinThan(dealer.cards.optimalScore())
    }
```
2. return을 if 문 밖으로 뺄 수 있겠네요. 여기서부터는 코드 스타일 차이라 꼭 반영하실 필요는 없지만 저는 코틀린에서 세 가지 이상 조건문이 분기되는 경우 when이 가독성면에서 더 좋다고 생각하여 다음처럼 작성하는걸 선호합니다.

```suggestion
    fun isWinThan(that: Score): Boolean {
        return when {
            isBust() -> false
            that.isBust() -> true
            else -> value >= that.value
        }
    }

```